cao_van_thao/lesson3/currency-exchange.php

```diff
@@ -0,0 +1,33 @@
+<?php
+    class change{
+        public $InputMoney;

```

Comment:
```suggestion
        public $inputMoney;
```
Tên biết bắt đầu bằng chữ in thường e ơi